spec/book_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../book'
+
+describe Book do
+  context 'Unit Test For Book Class' do
+    before(:context) do
+      title = 'Mastery'
+      author = 'Robert Greene'
+      @book = Book.new(title, author)
+    end
+    it 'If Book Class creates a new instance.' do
+      expect(@book).to be_an_instance_of(Book)
+    end
+    it 'If Book Class has a book title' do
+      expect(@book.title).to eq('Mastery')
+    end
+    it 'If Book Class has a book author' do
+      expect(@book.author).to eq('Robert Greene')
+    end
+  end
```

Comment:
- [ ] Good job with test coverage, but since we were aiming for 100% coverage, this is to make sure our app does not break in production, and it a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)for this project,  "`Add unit tests for all the methods in all the classes`", please it would be nice to cover the following methods is the `Book Class` with tests: 

- `add_rental`: This method should return a rental and add it to the book.  Something like so: 
```
  context "When adding a new rentals" do

    it "The add_rental method returns a rental and adds it to the book" do
      book = Book.new("Title", "Author")
      person = Person.new(24, "Ariel")

      rental = book.add_rental(person, "2021-05-04")

      expect(book.rentals).to eql [rental]
    end
  end
```
Please kindly fix that